src/service/config/configuration-service.ts

```diff
@@ -109,4 +131,49 @@ export class ConfigurationService {
     }
     return {};
   }
+
+  /**
+   * Returns the information for the target repository. For PR flow types it is the base branch
+   * @returns
+   */
+  getTargetProject(): ProjectConfiguration {
+    return this.configuration.targetProject;
+  }
+
+  /**
+   * Returns the information for the source repository. For PR flow types it is the head branch
+   * @returns
+   */
+  getSourceProject(): ProjectConfiguration {
+    return this.configuration.sourceProject;
+  }
+
+  /**
+   * Root folder is outputFolder if defined via options otherwise GITHUB_WORKSPACE.
+   * If both of these are undefined, it is the current working directory
+   * @returns
+   */
+  getRootFolder(): string {
+    const rootFolder = this.configuration.parsedInputs.outputFolder ?? process.env.GITHUB_WORKSPACE;
```

Comment:
```suggestion
    return this.configuration.parsedInputs.outputFolder ?? process.env.GITHUB_WORKSPACE ?? process.cwd();
```
and to remove next line